README.md

```diff
@@ -22,7 +22,7 @@ The rest are optional:
 |```LOCAL_NETWORK=192.168.1.0/24```|Whether to route and allow input/output traffic to the LAN. LAN access is blocked by default if not specified. Multiple ranges can be specified, separated by a comma or space. Note that there may be DNS issues if this overlaps with PIA's default DNS servers (`10.0.0.243` and `10.0.0.242` as of July 2022). Custom DNS servers can be defined using `VPNDNS` (see below) if this is an issue.
 |```KEEPALIVE=25```|If defined, PersistentKeepalive will be set to this in the WireGuard config.
 |```MTU=1420```|This can be used to override ```wg-quick```'s automatic MTU setting on the Wireguard interface if needed. By default this remains unset (ie. let ```wg-quick``` choose).
-|```VPNDNS=8.8.8.8, 8.8.4.4```|Use these DNS servers in the WireGuard config. Defaults to PIA's DNS servers if not specified.
+|```VPNDNS=8.8.8.8, 8.8.4.4```|Use these DNS servers in the WireGuard config. PIA's DNS servers will be used if not specified. Use 0 to disable making any changes to the default container DNS settings. (Note: Using any DNS servers other than PIA's may lead to DNS queries being leaked outside the VPN connection.)

```

Comment:
I think that leaky DNS queries might only be an issue when `VPNDNS=0` is used. My understanding is that DNS queries should still go down the VPN tunnel when custom servers are used (unless it happens to fall in the `LOCAL_NETWORK` range which has to manually be set and is expected to bypass the tunnel anyway). However when sticking with the Docker defaults (ie. `VPNDNS=0`) then queries may leak out via the host machine though Docker's embedded DNS server.

From [here:](https://docs.docker.com/config/containers/container-networking/#dns-services) 

> By default, containers inherit the DNS settings of the host, as defined in the /etc/resolv.conf configuration file. Containers that attach to the default bridge network receive a copy of this file. Containers that attach to a custom network use Docker’s embedded DNS server. The embedded DNS server forwards external DNS lookups to the DNS servers configured on the host.

```suggestion
|```VPNDNS=8.8.8.8, 8.8.4.4```|Use these DNS servers in the WireGuard config. PIA's DNS servers will be used if not specified. Set to 0 to disable making any changes to the default container DNS settings, however doing so may lead to DNS queries being leaked outside the VPN connection.
```

Reply:
That sounds fair. It might be better to stick with the broader definition then.

Thanks again for the PR.